app/src/main/java/org/sopt/and/screens/LoginActivity.kt

```diff
@@ -0,0 +1,273 @@
+                        containerColor = WavveTheme.colors.VividBlue
+                    ),
+                    shape = RoundedCornerShape(size = 50.dp),
+                    onClick = {
+                        if (email == localEmail && password == localPassword) {
+                            //로그인 성공
+                            Intent(context, MyActivity::class.java).apply {
+                                putExtra(EMAIL, email)
+                                flags =
+                                    Intent.FLAG_ACTIVITY_CLEAR_TASK or Intent.FLAG_ACTIVITY_NEW_TASK
+                                context.startActivity(this)
+                            }
+                        } else {
+                            //로그인 실패
+                            scope.launch {
+                                snackbarHostState.showSnackbar("로그인에 실패했습니다.")
+                            }
+                        }
+
+                        //키보드 내리기
+                        keyboardController?.hide()
+                    },
+                ) {
+                    Text(
+                        text = stringResource(R.string.Login),
+                        color = Color.White
+                    )
+                }
+
+                Spacer(modifier = Modifier.height(20.dp))
+
+                Box(
+                    modifier = Modifier.fillMaxWidth(),
+                    contentAlignment = Alignment.Center
+                ) {
+                    Row(
+                        horizontalArrangement = Arrangement.spacedBy(10.dp),
+                        verticalAlignment = Alignment.CenterVertically,
+                        modifier = Modifier.height(IntrinsicSize.Min)
+                    ) {
+                        Text(
+                            stringResource(R.string.FindID),
+                            fontSize = 12.sp,
+                            color = Color.Gray,
+                        )
+                        VerticalDivider(
+                            thickness = 1.dp,
+                            color = Color.DarkGray
+                        )
+                        Text(
+                            stringResource(R.string.ResetPassword),
+                            fontSize = 12.sp,
+                            color = Color.Gray,
+                        )
+                        VerticalDivider(
+                            thickness = 1.dp,
+                            color = Color.DarkGray
+                        )
+                        Text(
+                            stringResource(R.string.SignUp),
+                            fontSize = 12.sp,
+                            color = Color.Gray,
+                            modifier = Modifier.clickable {
+                                Intent(context, SignUpActivity::class.java).apply {
+                                    context.startActivity(this)
+                                }
```

Comment:
apply는 객체를 수정할 때 사용하는 스코프 함수입니다. 이 보다는
```suggestion
                                Intent(context, SignUpActivity::class.java).let { intent ->
                                    context.startActivity(intent)
                                }
```